ci/tox/local-run.sh

```diff
@@ -8,18 +8,20 @@ function check_var() {
   grep -q "^$1=" .env || { echo "Var $1 missing both from environment and .env file"; return 1; }
 }
 
```

Comment:
Hmm generally `ci/tox/local-run.sh` can probably be retained indeed... just with the remark about the reusable functions